src/auth-service/services/rbac.service.js

```
@@ -1084,6 +1267,221 @@ class RBACService {
+
+      // Manually populate the user data
+      const populatedUser = await this._populateUserRoleData(user);
+
+      const groupRoles = populatedUser.group_roles || [];
+      const networkRoles = populatedUser.network_roles || [];
+
+      const groupPermissions = new Set();
+      const networkPermissions = new Set();
+
+      groupRoles.forEach((groupRole) => {
+        if (groupRole.role && groupRole.role.role_permissions) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)